docs/platform_management_plan/project_management.rst

```diff
@@ -0,0 +1,165 @@
+   # Copyright (c) 2024 Contributors to the Eclipse Foundation
+   #
+   # See the NOTICE file(s) distributed with this work for additional
+   # information regarding copyright ownership.
+   #
+   # This program and the accompanying materials are made available under the
+   # terms of the Apache License Version 2.0 which is available at
+   # https://www.apache.org/licenses/LICENSE-2.0
+   #
+   # SPDX-License-Identifier: Apache-2.0
+   # *******************************************************************************
+
+Project management plan
+#######################
+
+Project organization
+====================
+
+Steering committees
+-------------------
+Steering of the project is done by two committees: *project lead circle* and *technical lead circle*.
+
+* **Project lead circle**
+
+  Members of *Project lead circle* are the project leads of the *SCORE* project. The election of project leads is done as described in the `Project Roles chapter <https://www.eclipse.org/projects/handbook/#roles-pl>`_ of *Eclipse Foundation Project Handbook*. In case of absence, a project lead can nominate a deputy.
+
+  The main tasks of the *Project lead circle* are:
+
+  * Definition, discussion of and decisions about strategical topics (e.g. which associations to approach, confirmation of roadmap, representation in public).
+  * Decision on which new software modules should be added or removed from the project. The decision is done based on proposal from the *Technical lead circle*. In case of changes to the existing modules and no concordant decision in the *Technical Lead Circle*, the *Project Lead Circle* has to decide about the change.
+  * Election of new Project Leads.
+  * Last instance of escalation path.
+
+  *Project lead circle* proposes and elects a *Project lead circle Assistant* and his deputy with bare majority, who is responsible for scheduling and announcing meetings, preparing and announcing agenda, writing meeting minutes and protocols. The *Project lead circle Assistant* and his deputy can resign anytime on their own will.
+
+* **Technical lead circle**
+
+  Each *Project Lead* is allowed to nominate one *Technical Lead*. The *Technical Leads* form the "Technical Lead Circle". In case of absence, a technical lead can nominate a deputy. *Technical Leads* have the following responsibilities:
+
+  * Review and approval of *Contribution Requests*
+  * Project management, e.g., creation of the roadmap
+  * High-level project control and coordination between multiple software modules.
+  * Escalation instance for software module project leads and committers.
+
+  *Technical lead circle* proposes and elects a *Technical lead circle Assistant* and his deputy with bare majority, who is responsible for scheduling and announcing meetings, preparing and announcing agenda, writing meeting minutes and protocols. The *Technical lead circle Assistant* and his deputy can resign anytime on their own will.
```

Comment:
We should also define how the Assistant will be elected and if there will be something like a reelection after some time